applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c

```diff
@@ -115,6 +129,71 @@ static void le_audio_event_publish(enum le_audio_evt_type event, struct bt_conn
 	ERR_CHK(ret);
 }
 
+/**
+ * @brief	Add a conn pointer to the discover list
+ *
+ * @param	conn	Pointer to the connection to add
+ * @param	dir	Directions to do discovery for
+ *
+ * @retval	-ENOMEM	No more space for connections
+ * @retval	0	Successfully added connection
+ */
+static int discover_list_add(struct bt_conn *conn, enum unicast_discover_dir dir)
```

Comment:
```suggestion
static int discover_list_add(struct bt_conn const * const conn, enum unicast_discover_dir dir)
```

Reply:
The first const will be discarded below, so will change to:
```suggestion
static int discover_list_add(struct bt_conn *const conn, enum unicast_discover_dir dir)
```